frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx

```diff
@@ -28,7 +31,20 @@ const FacilityInformation: React.FC<Props> = ({
   };
 
   return (
-    <div className="usa-form usa-form--large">
+    <div className={classNames("usa-form", "usa-form--large")}>
```

Comment:
what does using `classNames` do? when I inspect the elements, it seems like they get rendered the same without it

Reply:
oh interesting - https://github.com/JedWatson/classnames
this might be a nit but it doesn't look like we need it here though I don't think

Reply:
Yeah, doesn't look like it's useful here, I think this is some way outdated React knowledge peeking through on my end.

`classnames` isn't totally useless but from what I can see it's most helpful to conditionally add CSS classes to an element:

https://github.com/CDCgov/prime-simplereport/blob/fec010bbfe37a5fa7ddf37bd82c3e0e221aa83b5/frontend/src/app/commonComponents/Dropdown.tsx#L62-L66

Cleaned up in [69fc240](https://github.com/CDCgov/prime-simplereport/pull/4433/commits/69fc2402831b6e640bf1a0ec203ff12f04066d69).